src/posts/misconceptions-about-majority-rule.mdx

```diff
@@ -15,38 +15,41 @@ categories:
 tags:
     - voting
 ---
+import { resolvePath } from '@/utils/helpers.js'
 
-<!--
+{/*
 FYI, this is another example of academia gone wrong:
 http://www-groups.dcs.st-and.ac.uk/history/PrintHT/Voting.html
 
 - [ ] Post followup two days later.
 - [ ] post job listing to: https://govuejs.com/jobs/1/full-stack-engineer
--->
+*/}
```

Comment:
`<!-- ... -->` is an illegal syntax in `MDX` so turned it into a JSX equivalent, which is `/* ... */`
reference: [here](https://mdxjs.com/docs/what-is-mdx/#mdx-syntax)